src/v/kafka/server/handlers/fetch.h

```diff
@@ -327,6 +327,7 @@ struct read_result {
     model::partition_id partition;
     std::vector<cluster::rm_stm::tx_range> aborted_transactions;
     memory_units_t memory_units;
+    bool read_from_follower;
```

Comment:
Default initialization would be nice. Also, where this flag is actually initialized? I can't find it.

Reply:
i need to remove it as it is not used anywhere